WordPress/Classes/ViewRelated/Site Creation/Site Name/SiteNameView.swift

```diff
@@ -144,14 +144,14 @@ private extension SiteNameView {
     func setupTitleColors() {
         // enclose the vertical name between two characters that are not in the title
         // (and reasonably never will..) to distinguish it from any substring in the title
-        let selectedVerticalName = "😎" + siteVerticalName + "🙃"
+        let selectedVerticalName = "😎" + siteVerticalName.dropFirst().dropLast() + "🙃"
```

Comment:
This is fascinating, and to me a little scary. 😆 But, it works for the purpose of this PR!

If we really only care about the range to do the substitution, another solution could be something like this.

```
    /// Highlights the site name in blue
    func setupTitleColors() {
        let replacementIndex = NSString(string: TextContent.title).range(of: "%@")
        guard !siteVerticalName.isEmpty, replacementIndex.length > 0 else {
            titleLabel.setText(TextContent.defaultTitle)
            return
        }

        let fullTitle = String(format: TextContent.title, siteVerticalName)
        let attributedTitle = NSMutableAttributedString(string: fullTitle)
        let replacementRange = NSRange(location: replacementIndex.location, length: siteVerticalName.utf16.count)

        attributedTitle.addAttributes([
            .foregroundColor: UIColor.primary,
        ], range: replacementRange)
        titleLabel.attributedText = attributedTitle
    }
```

`siteVerticalName.utf16.count` actually solves a little emoji bug due to how Swift strings treat characters.

Inputting a custom vertical name like ✌️😄😄😭☺️ (this was purely random on my part, a more efficient version could be constructed) results in the following:

| Input | Output |
| - | - |
| ![Simulator Screen Shot - iPhone 13 - 2022-04-20 at 15 41 33](https://user-images.githubusercontent.com/2092798/164311506-f0791fb3-4016-40f7-841a-4ffaa6bf211b.png) | ![Simulator Screen Shot - iPhone 13 - 2022-04-20 at 15 43 07](https://user-images.githubusercontent.com/2092798/164311702-9688ff8f-afd2-49be-a8a4-13abb4d3b157.png) |

Using `utf16.count` will fix it:

<img src="https://user-images.githubusercontent.com/2092798/164311785-ee13c9f0-0937-4475-9ffe-e463e5124fce.png" width="350px" alt="After the emoji fix" />

Reply:
Awesome suggestion @twstokes ! I like your solution much better than mine; just pushed those changes. Thank you!